spec.html

```diff
@@ -4652,16 +4663,19 @@ <h1>The PrivateElement Specification Type</h1>
       </emu-table>
     </emu-clause>
 
-    <emu-clause id="sec-classfielddefinition-record-specification-type">
-      <h1>The ClassFieldDefinition Record Specification Type</h1>
-      <p>The ClassFieldDefinition type is a Record used in the specification of class fields.</p>
-      <p>Values of the ClassFieldDefinition type are Record values whose fields are defined by <emu-xref href="#table-classfielddefinition-fields"></emu-xref>. Such values are referred to as <dfn variants="ClassFieldDefinition Record">ClassFieldDefinition Records</dfn>.</p>
-      <emu-table id="table-classfielddefinition-fields" caption="ClassFieldDefinition Record Fields">
+    <emu-clause id="sec-classelementdefinition-record-specification-type" oldids="sec-classfielddefinition-record-specification-type">
+      <h1>The ClassElementDefinition Record Specification Type</h1>
+      <p>The ClassElementDefinition type is a Record used in the specification of class element definitions.</p>
```

Comment:
The definition repeats itself (The FooBarBaz [...] is a foo bar baz). Maybe it should be detalized as in `The PrivateElement type is a Record used in the specification of private class fields, methods, and accessors.`